astrosat_users/tests/utils.py

```diff
@@ -19,8 +20,7 @@ def generate_password(**kwargs):
     }
     password_kwargs.update(kwargs)
     assert password_kwargs["length"] >= 4  # faker will break if the pwd is _too_ short
-    password_faker = FactoryFaker("password", **password_kwargs)
-    return password_faker.generate(extra_kwargs={})
+    return fake.password(**password_kwargs)
```

Comment:
I notice the value of `length` is set to `>= 4`, didn't know you can do that, thought it was just a dictionary of values. I don't get how this works, can you point me towards the docs for the `password` function as I don't see anything that states you can do this.

Reply:
It's not _set to_ `>= 4`.  It's just an `assert` statement making sure that length is greater than 4.

There was some low-level code we were using that complained if the password was too short.

Reply:
Do'h, didn't see the `assert` stupid of me, sorry. Why is the check necessary since you are setting the kwargs value of `length` to 20. If this is only ever used in the tests, it's not possible to be less than 20 chars. I take it `length` means, generate a password of length X.

Reply:
b/c I could potentially overwrite any one of `password_kwargs` in line 21:

```
    password_kwargs.update(kwargs)
```